chainstate/src/detail/transaction_verifier/mod.rs

```diff
@@ -492,23 +509,23 @@ impl<'a, S: BlockchainStorageRead> CachedInputs<'a, S> {
 
             prev_tx_index_op
                 .spend(outpoint.output_index(), spender.clone())
-                .map_err(StateUpdateError::from)?;
+                .map_err(ConnectTransactionError::from)?;
         }
 
         Ok(())
     }
 
-    pub fn spend(
+    pub fn connect_transaction(
```

Comment:
Maybe `apply_spend` may be now renamed to `spend`

Reply:
Will do